lib/modules/manager/sveltos/__fixtures__/validEventTrigger.yml

```diff
@@ -0,0 +1,48 @@
+---
+apiVersion: lib.projectsveltos.io/v1beta1
+kind: EventTrigger
+metadata:
+  name: baseline
+spec:
+  helmCharts:
+  - repositoryURL:    https://prometheus-community.github.io/helm-charts
+    repositoryName:   prometheus-community
+    chartName:        prometheus-community/prometheus
+    chartVersion:     "23.4.0"
+  - repositoryURL:    https://kyverno.github.io/kyverno/
+    repositoryName:   kyverno
+    chartName:        kyverno/kyverno
+    chartVersion:     "v3.2.5"
+---
+apiVersion: lib.projectsveltos.io/v1beta1
+kind: EventTrigger
```

Comment:
Could people want to create rules specific to `EventTrigger`, `ClusterProfile` or `Profile`?

Reply:
It's probably more common to have them all combined instead of making specific rules. Should i combine the fixtures?

Reply:
I'm referring here to `depType`s. See this section of the `terraform` manager as an example. https://docs.renovatebot.com/modules/manager/terraform/#disabling-parts-of-the-manager